modules/gapi/include/opencv2/gapi/infer/parsers.hpp

```diff
@@ -46,57 +50,61 @@ namespace parsers {
 } // namespace nn
 
 /** @brief Parses output of SSD network.
+
 Extracts detection information (box, confidence, label) from SSD output and
 filters it by given confidence and label.
 
-@note Function textual ID is "org.opencv.core.parseSSD_BL"
+@note Function textual ID is "org.opencv.nn.parsers.parseSSD_BL"
 
 @param in Input CV_32F tensor with {1,1,N,7} dimensions.
-@param in_sz Size to project detected boxes to (size of the input image).
-@param confidence_threshold If confidence of the
+@param inSz Size to project detected boxes to (size of the input image).
+@param confidenceThreshold If confidence of the
 detection is smaller than confidence threshold, detection is rejected.
-@param filter_label If provided (!= -1), only detections with
+@param filterLabel If provided (!= -1), only detections with
 given label will get to the output.
 @return a tuple with a vector of detected boxes and a vector of appropriate labels.
 */
 GAPI_EXPORTS std::tuple<GArray<Rect>, GArray<int>> parseSSD(const GMat& in,
-                                                            const GOpaque<Size>& in_sz,
-                                                            const float confidence_threshold = 0.5f,
-                                                            const int   filter_label = -1);
+                                                            const GOpaque<Size>& inSz,
+                                                            const float confidenceThreshold = 0.5f,
+                                                            const int   filterLabel = -1);
```

Comment:
This version has no alignment to square right?

Reply:
Yes. Kernel with two outputs has alignment to square.

Reply:
Kernel with two outputs has label filtering.